ajax/libs/focusable/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "focusable",
+  "filename": "focus-element-overlay.min.js",
+  "version": "1.0.0",
+  "description": "Set focus on DOM element adding overlay to the rest of the page",
+  "homepage": "http://zzarcon.github.io/focusable/",
```

Comment:
Let's remove this. cdnjs hope "homepage" has at least link to its repo rather than demo.